src/AppInstallerCLI/AppInstallerCLI.vcxproj

```diff
@@ -136,6 +136,7 @@
       <PreprocessorDefinitions>_CONSOLE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <WarningLevel>Level4</WarningLevel>
       <AdditionalOptions>%(AdditionalOptions) /permissive- /bigobj</AdditionalOptions>
+      <MultiProcessorCompilation>true</MultiProcessorCompilation>

```

Comment:
This could also go in `/src/Directory.Build.props` so that you only need to add it in a single place. (I.e., what I wanted to do back in #3275 but I never finished...)

Reply:
I suspect that @florelis was suggesting that you put this property into the Directory.Build.props file that I did wire in to the existing branch already (without all of the other stuff that her change includes).